go/libkb/passphrase_helper.go

```diff
@@ -131,6 +131,14 @@ func GetPassphraseUntilCheck(m MetaContext, arg keybase1.GUIEntryArg, prompter P
 		if checker == nil {
 			return res, nil
 		}
+
+		s := res.Passphrase
+		t, err := checker.Automutate(m, s)
```

Comment:
I meant that instead of calling `checker.Automutate(...)` here you would call `paperKeyPrefixMutate(m, s, expectedPrefix)` and have that function in this `passphrase_helper.go` file.

Reply:
But how would this function know if it was prompting for a paperkey? Should that be passed in from the caller?

Reply:
nvm, I can inspect the gui arg...

Reply:
but then you still have to pass in the expectedPrefix right

Reply:
sorry, i thought this was in `GetPaperKeyPassphrase`....GH hid the rest...

Reply:
ok, we can leave it the way you have it.